examples/create-account-with-thresholdkey.js

```diff
@@ -0,0 +1,145 @@
+import {
+    Client,
+    PrivateKey,
+    AccountId,
+    Hbar,
+    AccountCreateTransaction,
+    TransferTransaction,
+    AccountBalanceQuery,
+    AccountDeleteTransaction,
+    KeyList,
+} from "@hashgraph/sdk";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+async function main() {
+    if (
+        process.env.OPERATOR_ID == null ||
+        process.env.OPERATOR_KEY == null ||
+        process.env.HEDERA_NETWORK == null
+    ) {
+        throw new Error(
+            "Environment variables OPERATOR_ID, HEDERA_NETWORK, and OPERATOR_KEY are required.",
+        );
+    }
+
+    const operatorId = AccountId.fromString(process.env.OPERATOR_ID);
+    const operatorKey = PrivateKey.fromStringED25519(process.env.OPERATOR_KEY);
+
+    // Create a client for the local Hedera network
+    const client = Client.forNetwork({
+        [process.env.HEDERA_NETWORK]: new AccountId(3)
+    });
+    client.setOperator(operatorId, operatorKey);
+
+    try {
+        console.log("Create Account With Threshold Key Example Start!");
+
+        /*
+         * Step 1:
+         * Generate three new Ed25519 private, public key pairs.
+         *
+         * You do not need the private keys to create the Threshold Key List,
+         * you only need the public keys, and if you're doing things correctly,
+         * you probably shouldn't have these private keys.
+         */
+
+        // Generate 3 new Ed25519 private, public key pairs
+        const privateKeys = [];
+        const publicKeys = [];
+        for (let i = 0; i < 3; i++) {
+            const key = PrivateKey.generateED25519();
+            privateKeys.push(key);
+            publicKeys.push(key.publicKey);
+        }
+
+        console.log("Generating public keys...");
+        publicKeys.forEach((publicKey, index) => {
+            console.log(`Generated public key ${index + 1}: ${publicKey.toString()}`);
+        });
+
+        /*
+         * Step 2:
+         * Create a Key List.
+         *
+         * Require 2 of the 3 keys we generated to sign on anything modifying this account.
+         */
+        const thresholdKey = new KeyList();
+        thresholdKey.setThreshold(2);
+        publicKeys.forEach((publicKey) => thresholdKey.push(publicKey));
```

Comment:
You can pass the publicKeys directly as a parameter to the KeyList.

Reply:
resolved